rpm/sofia-sip.spec

```diff
@@ -23,59 +26,59 @@ multimedia distribution, and multimedia conferences.
 
 
 %package glib
-Summary:    Glib bindings for Sofia-SIP
-Group:      Communications/Telephony and IM
+Summary:    Glib bindings for Sofia-SIP 
 Requires:   %{name} = %{version}-%{release}
-Requires(post): /sbin/ldconfig
-Requires(postun): /sbin/ldconfig
 
 %description glib
 GLib interface to Sofia SIP User Agent library.
 
 %package utils
 Summary:    Sofia-SIP Command Line Utilities
-Group:      Applications/Internet
 Requires:   %{name} = %{version}-%{release}
 
 %description utils
 Command line utilities for the Sofia SIP UA library.
 
 %package glib-devel
 Summary:    Glib bindings for Sofia SIP development files
-Group:      Development/Libraries
-Requires:   %{name} = %{version}-%{release}
-Requires:   sofia-sip-glib = %{version}-%{release}
-Requires:   sofia-sip-devel = %{version}-%{release}
+Requires:   %{name}-glib = %{version}-%{release}
+Requires:   %{name}-devel = %{version}-%{release}
+Requires:   pkgconfig
 
 %description glib-devel
 Development package for Sofia SIP UA Glib library. This package
 includes libraries and include files for developing glib programs
 using Sofia SIP.
 
-
 %package devel
 Summary:    Sofia-SIP Development Package
-Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
+Requires:   pkgconfig
 
 %description devel
 Development package for Sofia SIP UA library.
 
+%package doc
+Summary:   Documentation for %{name}
+Requires:  %{name} = %{version}-%{release}
+
+%description doc
+Documentation for %{name}.
+
+
 %prep
-%setup -q -n %{name}-%{version}/src
+%setup -q -n %{name}-%{version}/%{name}
 
 %build
-%configure --disable-static
+./autogen.sh
+%configure --disable-static --without-doxygen --disable-stun
 make %{?_smp_mflags}
 
 %install
-rm -rf %{buildroot}
 %make_install
-
+find %{buildroot} -name \*.la -delete
```

Comment:
This is not needed rpmbuild already does it.

Reply:
Dropped it